portal/src/main/scala/cromwell/pipeline/controller/UserController.scala

```diff
@@ -1,26 +1,39 @@
 package cromwell.pipeline.controller
 
+import akka.http.scaladsl.server.Directives.{ complete, concat, get, onComplete, parameter, path }
+
 import akka.http.scaladsl.model.StatusCodes
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.server.Route
 import cromwell.pipeline.datastorage.dto.UserId
 import cromwell.pipeline.service.UserService
 import cromwell.pipeline.utils.auth.AccessTokenContent
-import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport
 
 import scala.concurrent.ExecutionContext
 import scala.util.{ Failure, Success }
 
-class UserController(userService: UserService)(implicit executionContext: ExecutionContext) {
+class UserController(userService: UserService)(implicit executionContext: ExecutionContext) extends PlayJsonSupport {
```

Comment:
I guess in this case it is better to have composition instead of inheritance.

Reply:
solved it with `import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._`